tests/test_response.py

```diff
@@ -1052,7 +1052,7 @@ def dummy_wsgi_callable(environ, start_response):
     environ = {}
 
     def dummy_start_response(status, headers, exc_info=None):
-        assert headers, [("Set-Cookie" == "a=1; Path=/")]
+        assert headers, ["Set-Cookie" == "a=1; Path=/"]
```

Comment:
Is this missing `==` ? 🤔

Reply:
Not sure what happened here.  I didn't touch it; the formatter picked it up.

Reply:
Removing unnecessary parens, which is ok. But what was the intent of the assertion before this change? Interpreting this line as «assert headers is not empty, otherwise print this list as debug message» seems less useful to me than «assert headers match this cookie» !